src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1250,6 +1269,42 @@ public static void selectAndDeleteTextInJavaPart(RemoteRobot remoteRobot, String
         TestUtils.sleepAndIgnoreException(2);
     }
 
+    /**
+     * Deletes a string of text from the currently focused editor
+     *
+     * @param remoteRobot      The RemoteRobot instance.
+     * @param fileName         The fileName to modify
+     * @param textToModify     The string to modify
+     * @param modificationText The string to use for modification
+     */
+    public static void selectAndModifyTextInJavaPart(RemoteRobot remoteRobot, String fileName, String textToModify, String modificationText){
+        clickOnFileTab(remoteRobot, "server.xml");
```

Comment:
this doesn't seem right... should it be `fileName` passed in the method?

Reply:
yes, fixed